composeApp/src/commonMain/kotlin/com/arkivanov/minesweeper/settings/DefaultEditSettingsComponent.kt

```diff
@@ -40,13 +44,11 @@ internal class DefaultEditSettingsComponent(
         val height = _model.value.height.toIntOrNull() ?: return
         val maxMines = _model.value.maxMines.toIntOrNull() ?: return
 
-        onConfirmed(
-            GameSettings(
-                width = width,
-                height = height,
-                maxMines = maxMines,
-            )
-        )
+        val finalWidth = width.coerceIn(MIN_SIZE..MAX_WIDTH)
+        val finalHeight = height.coerceIn(MIN_SIZE..MAX_HEIGHT)
+        val finalMines = maxMines.coerceIn(1 until finalWidth * finalHeight - 1)
```

Comment:
I think `until` is exclusive on the right side. We could write `coerseIn(1, finalWidth * finalHeight - 1)`, or `coerseIn(1 until finalWidth * finalHeight)`.